canvas_modules/common-canvas/src/themes/light.scss

```diff
@@ -14,15 +14,4 @@
  * limitations under the License.
  */
 
-@use "@carbon/react/scss/themes" as *;
-@use "../carbon.scss" as *;
-
-// Switch the themes below to get light or dark.
-$theme: $g10; // The light (gray 10) theme
-// $theme: $g90; // The dark theme
-
-:root {
-	@include theme();
-}
-
-@import "../index.scss";
+@use "../index.scss";
```

Comment:
@nmgokhale Well done! - you removed the code -- I just tried deleting the file. 
Since we are down to just one line here, is it possible to get rid of the file completely?